tests/rptest/scale_tests/many_topics_test.py

```diff
@@ -1330,6 +1471,11 @@ def test_many_topics_config(self):
             # heap space on such a large request.
             pass
 
+        # If the test exits right after this the controll log size will trigger
+        # an oversized error. Hence we give the controller some time to compact
+        # here.
+        time.sleep(60)
```

Comment:
All these sleeps are really not great, is there some metric we can query to wait on?

Reply:
Also feels like a bug we should fix?

Reply:
It's not causing an oversized alloc. There is a separate check in DT for the size of the controller on disk that is falling.

Reply:
Ah I see interesting.

Reply:
Other option is to add a way to disable this check for this test, or to call an API which forces an immediate compaction, because 60 second sleep just sounds like a recipe for flakiness. You can for advice too from whoever added this check.

Reply:
@ballard26 what was the resolution or further comment here?